transactions/read-only/parallel.md

```diff
@@ -23,112 +23,115 @@ Chain APIs can be classified into reads and writes.
+| schedule_protocol_feature_activations      | medium_high |                              | no                     | no                       |
+| get_supported_protocol_features            | medium_high |                              | yes                    | yes                      |
+| get_account_ram_corrections                | medium_high |                              | yes                    | yes                      |
+| get_unapplied_transactions                 | medium_high |                              | yes                    | yes                      |
 
 ### Net APIs
 Net APIs do modify state but do not modify the state accessible to read-only transaction execution. They are received on the HTTP thread, processed on the main thread, and responses are sent on the HTTP thread.
 
-| API                         | priority    | Global data modified           | Safe to read-only trx? |
-|-----------------------------|-------------|--------------------------------|------------------------|
-| connect                     | medium_high | main thread: net::connections  | yes                    |
-| disconnect                  | medium_high | main thread: net::connections  | yes                    |
-| status                      | medium_high | none                           | yes                    |
-| connections                 | medium_high | none                           | yes                    |
+| API                         | priority    | Global data modified           | Safe to read-only trx? | Safe to run in parallel? |
+|-----------------------------|-------------|--------------------------------|------------------------|--------------------------|
+| connect                     | medium_high | main thread: net::connections  | yes                    | no                       |
+| disconnect                  | medium_high | main thread: net::connections  | yes                    | no                       |
+| status                      | medium_high | none                           | yes                    | yes                      |
+| connections                 | medium_high | none                           | yes                    | yes                      |
 
 
 ### Trace APIs
 Trace APIs do not go through main thread. They are executed only in http thread.
-| API                         | priority |  Global data modified  | Safe to read-only trx? |
-|-----------------------------|----------|------------------------|------------------------|
+| API                         | priority |  Global data modified  | Safe to read-only trx? | Safe to run in parallel? |
+|-----------------------------|----------|------------------------|------------------------|--------------------------|
 | get_block                   | NA       | none                   | yes                    |
 | get_transaction_trace       | NA       | none                   | yes                    |
```

Comment:
These are both yes for safe to run in parallel as they do not use the main thread at all.